README.md

```diff
@@ -14,49 +13,54 @@ public cloud.
 
 *CLOUD FOUNDRY and OPEN SERVICE BROKER are trademarks of the CloudFoundry.org Foundation in the United States and other countries.*
 
-## Supported Services
+## Supported services
```

Comment:
We'd prefer to stick with title case for the section headings, i.e. "Supported Services"